src/stylesheets/components/_np-score.scss

```
@@ -0,0 +1,76 @@
+.np-score {
+  margin-top: rem(45px);
+
+  text-align: center;
+
+  &__number-container {
+    width: fit-content;
+    height: rem(56px);
+    margin: 0 auto;
+    padding: 0;
+
+    list-style: none;
+
+    border: 0.5px solid map-get($brand-white, 'primary');
+    border-radius: rem(12px);
+  }
+
+  &__number-label {
+    display: inline-block;
+    height: rem(55px);
+    margin: 0;
+
+    border-right: 0.5px solid map-get($brand-white, 'primary');
+  }
+
+  &__number-label--no-border {
+    border-right: 0;
+  }
```

@rafayet-monon How about using the `:last-of-type` instead ? so we can remove the class `&__number-label--no-border`
```suggestion
  &__number-label:last-of-type {
    border-right: 0;
  }
```